.env.sample

```
@@ -1,2 +1,9 @@
-PORT=3000
-MONGO_URL=
+PORT= 3000
+MONGO_URL=<YOUR_MONGO_DB_URL>
+HOST=<YOUR_LOCAL_HOST_URL>
+DOMAIN=https://<YOUR_AUTH0_DOMAIN>
+CLIENTID=<YOUR_AUTH0_CLIENT_ID>
+CLIENTSECRET=<YOUR_AUTH0_CLIENT_SECRET>
+REDIRECTURI=<YOUR_CALLBACK_URL>
```

```suggestion
CLIENT_ID=<YOUR_AUTH0_CLIENT_ID>
CLIENT_SECRET=<YOUR_AUTH0_CLIENT_SECRET>
REDIRECT_URI=<YOUR_CALLBACK_URL>
```

however, those are not the same name as in your course, is that wanted?

---

No, it's not: I will wait for [this related PR](https://github.com/fewlinesco/sparta_courses/pull/908) to be merged for changes here.